dynamic_example/modules/custom/dynamic_example_homepage/plugins/content_types/cards/cards.inc

```diff
@@ -34,27 +34,37 @@ function dynamic_example_homepage_cards_content_type_render($subtype, $conf, $ar
     return;
   }
 
-  $cards = array();
+  // Holds the cards Html.
+  $cards_item = array();
 
   $nids = array_keys($result['node']);
+
   foreach (node_load_multiple($nids) as $node) {
 
     $wrapper = entity_metadata_wrapper('node', $node);
     $image = $wrapper->field_company_logo->value();
 
     $image = file_create_url($image['uri']);
 
+    $comment = format_plural($node->comment_count, '1 comment', '@count comments');
```

Comment:
$comment is a string here. If you want to check the comment count, you need to use the `IF` on the `$node->comment_count`